### ISSUE TITLE
Update wasmtime to the newest version (0.35)

### ISSUE THREAD

**Issue by Mossaka** [fermyon/spin#348]

**Comment by radu-matei** [comment]
There is a dependency graph that has to be updated before we can do this:

- [x] `wit-bindgen` — https://github.com/bytecodealliance/wit-bindgen/pull/192
- [ ]  `wasi-experimental-http` — https://github.com/deislabs/wasi-experimental-http
- [ ] Wagi — https://github.com/deislabs/wagi

I will open PRs to each of those and update the issue as we make progress.

Thanks!

**Comment by Mossaka** [comment]
`wasi-experimental-http` update to wasmtime 0.35
https://github.com/deislabs/wasi-experimental-http/pull/93

**Comment by Mossaka** [comment]
https://github.com/fermyon/spin/pull/466

**Comment by vdice** [comment]
Closing this one as no longer relevant; thank you!
